templates/deploy-ansible-service-broker.template.yaml

```diff
@@ -197,18 +218,45 @@ objects:
         recovery: ${RECOVERY}
         ssl_cert_key: /etc/tls/private/tls.key
         ssl_cert: /etc/tls/private/tls.crt
-        auth:
-          - type: basic
-            enabled: ${ENABLE_BASIC_AUTH}
+
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: ansibleservicebroker-client
+    namespace: ansible-service-broker
+
+- apiVersion: authorization.openshift.io/v1
+  kind: ClusterRoleBinding
+  metadata:
+    name: ansibleservicebroker-client
+  subjects:
+  - kind: ServiceAccount
+    name: ansibleservicebroker-client
+    namespace: ansible-service-broker
+  roleRef:
+    kind: ClusterRole
+    name: access-asb-role
+
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: ansibleservicebroker-client
+    annotations: 
+      kubernetes.io/service-account.name: ansibleservicebroker-client
+  type: kubernetes.io/service-account-token
 
 - apiVersion: servicecatalog.k8s.io/v1alpha1
   kind: ${BROKER_KIND}
   metadata:
     name: ansible-service-broker
   spec:
-    url: ${ASB_SCHEME}://asb-1338-ansible-service-broker.${ROUTING_SUFFIX}
+    url: https://asb.ansible-service-broker.svc:1338/ansible-service-broker/
```

Comment:
This makes it a requirement to use https.  I'm not against that, but then ASB_SCHEME should be removed. 

Another way to do this is keep ASB_SCHEME and BROKER_AUTH in place but change BROKER_AUTH to:
```yaml
value: '{ "bearer": { "secretRef": { "kind": "Secret", "namespace": "ansible-service-broker", "name": "ansible-service-broker" } } }'
```

Reply:
You also aren't using ```BROKER_URL_PREFIX```